### DISCUSSION TITLE
detect node runtime error

### DISCUSSION THREAD

**Discussion by NextPage66** [mtxr/vscode-sqltools#1358] [General]
I am new to this so obviously done something wrong.  
Everytime i start up VS Code i get this message..."Check Terminal view for an erroring 'detect node runtime' session".
any help?
![Node error-1](https://github.com/user-attachments/assets/da0d70db-4945-48a9-aac3-9c7e0f3e946d)
![Node error-2](https://github.com/user-attachments/assets/3e6ffd59-ed15-4e16-ab2f-d159c2eace1c)
![Node error-3](https://github.com/user-attachments/assets/955ac7b7-3123-4af1-8a28-b29330949d13)

**Comment by gjsjohnmurray** [comment]
Do you want to connect to SQLite databases? If yes, maybe you haven't met the [prerequisite](https://vscode-sqltools.mteixeira.dev/en/drivers/sq-lite) of having installed Node.js.

If you don't want to use the SQLite driver I recommend you set `sqlTools.useNodeRuntime` to `false`.

**Reply by NextPage66** [reply]
we don't have SQLite.  How do i set the runtime to false.

**Reply by Yahud100** [reply]
dear @gjsjohnmurray  is sqlTools.useNodeRuntime should be true for only sqlite or oracle as well?

**Reply by gjsjohnmurray** [reply]
I don't know about the Oracle driver as it's not part of the core project. I suggest you do some experiments with that setting.

**Comment by gjsjohnmurray** [comment]
See https://code.visualstudio.com/docs/getstarted/settings for documentation about how to edit settings.

**Reply by NextPage66** [reply]
thanks
